submit-web/src/components/DocumentUpload/PendingDocumentRow.tsx

```diff
@@ -0,0 +1,59 @@
+import { CircularProgress, Link as MuiLink, Typography } from "@mui/material";
+import { BCDesignTokens } from "epic.theme";
+import { PackageTableRow, StyledTableCell } from "./DocumentTableRow";
+
+type DocumentTableRowProps = {
+  documentItem: {
+    id: number;
+    name: string;
+  };
+  error?: boolean;
+};
+export default function PendingDocumentRow({
```

Comment:
how is the document being uploaded here? I don't see a save object to s3 call

Reply:
fixing that right now all the other comments have been adressed, it was working before i added this

Reply:
fixed